PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Application/Dtos/Doctor/RegisterDoctorDto.cs

```diff
@@ -0,0 +1,11 @@
+namespace PrivateHospitals.Application.Dtos.Doctor;
+
+public class RegisterDoctorDto
+{
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string UserName { get; set; }
+    public string Email { get; set; }
+    public string Speciality { get; set; }
+    public string Password { get; set; }
+}
```

Comment:
Rework all your DTOs to be `record` because it fits DTO pattern better. it has `init required` setters and records are immutable by design, so DTOs also should be immutable

Reply:
Fixed